src/seed_data/seed.ts

```diff
@@ -0,0 +1,51 @@
+import { Connection, createConnection, getConnection } from "typeorm";
+import { runSeeder, useSeeding } from "typeorm-seeding";
+import { Course } from "../entity/Course";
+import { Exam } from "../entity/Exam";
+import { Note } from "../entity/Note";
+import { Student } from "../entity/Student";
+import { StudentCourse } from "../entity/StudentCourse";
+import { StudentExam } from "../entity/StudentExam";
+import CreateCourses from "./seeds/createCourses.seed";
+import CreateStudents from "./seeds/createStudents.seed";
+
+import * as ora from 'ora';
+
+createConnection().then(async () => {
+    const spinner = ora();
+    spinner.start('Setting up seeding utilities');
+    await useSeeding().then((_) => spinner.succeed());
+
+    const connection: Connection = getConnection();
```

Comment:
with the new `ormconfig.json` stuff, would the fake generation still work since there's two connection options?
i did this in my chongus test setup pr when for running `npm start`
```
const connectionOptions = await getConnectionOptions("dev");
await createConnection({...connectionOptions, name: "default"}).then(
```

Reply:
oh you're right, I haven't set up your testing stuff yet -- let me do that and then I'll comment back here if it works/commit some changes if it breaks

Reply:
gooooood shout, that did break it, but I pushed some changes that should resolve!